### PR TITLE
Re-add missing settings

### LINKED ISSUE #519: Some VS settings are missing

### PR DESCRIPTION
**Pull request by GnomedDev**
This re-adds the settings from vanilla that were missing from the new settings menu, which closes #519.

I have no idea how to test these work, but I can't imagine how they would not.

### CODE REVIEW

**Comment by riidefi** [comment]
> I have no idea how to test these work
In the "Versus" game mode, click the blue Rule settings button in the upper right corner.

**Comment by GnomedDev** [comment]
I don't know how to tell the difference between CPU behavior on different difficulties or item chances.

**Comment by GnomedDev** [comment]
We got rid of the Single/Team Versus page.

**Review by riidefi — APPROVED** [review]

File: payload/sp/settings/ClientSettings.hh [hunk 1]
```diff
@@ -106,6 +110,12 @@ enum class EngineClass {
     CC100,
 };
 
+enum class CPUMode : u32 {
+    Easy = 0,
+    Normal = 1,
+    Hard = 2,
+};
```
[the remark this answers is not in the document]
Comment (GnomedDev) [reply payload/sp/settings/ClientSettings.hh:117]: Yes, I didn't see the point of VS without CPUs tbh, could re-add though.

[the remark this answers is not in the document]
Comment (Zachruff) [reply payload/sp/settings/ClientSettings.hh:117]: In the vanilla game the None option is only selectable with multiple local people (which then it makes sense). It should be re-added. Removing options from vanilla game is dummy.

[the remark this answers is not in the document]
Comment (GnomedDev) [reply payload/sp/settings/ClientSettings.hh:117]: It should be re-added when a better settings UI is developed, so it doesn't clutter the single options

**Review by stblr — APPROVED** [review]

**Pull request merged**